inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/fused_conv_eltwise_gpu_imad.cl

```diff
@@ -113,54 +170,114 @@ KERNEL (fused_convolution_eltwise_gpu_imad)(
-        weight_addr += SIMD_SIZE*NUM_FILTERS;
-#else
-        for(int pf=0; pf < NUM_FILTERS; pf++) {
-            w[pf] = weights[weight_addr];
-            weight_addr += SIMD_SIZE;
+                const bool input_on_padding = (((x_idx < 0) || (x_idx >= INPUT0_SIZE_X)) ||
+                                               ((y_idx < 0) || (y_idx >= INPUT0_SIZE_Y)));
+            #endif
+
+            #if INPUT0_LAYOUT_B_FS_YX_FSV16
+                #if ((FILTER_GROUPS_NUM > 1) && (FILTER_IFM_NUM % PACK != 0))
+                    #ifdef SHOULD_USE_DATA_ZP
+                        if (input_on_padding) {
+                            in[reg] = data_zp_val;
+                        } else {
+                    #endif
+                    INPUT0_TYPE* input_int8_arr = (INPUT0_TYPE*) &in[reg];
+                    in_addr = in_start_addr + reg * INPUT0_Y_PITCH * FSV;
+                    for (uint v = 0; v < PACK; v++) {
+                        int f_addr = ((feature_location + v) / FSV + INPUT0_PAD_BEFORE_FEATURE_NUM / FSV) * INPUT0_FEATURE_PITCH * FSV  + (feature_location + v) % FSV;
+                        input_int8_arr[v] = conv_input[in_addr + f_addr];
+                    }
+                    #ifdef SHOULD_USE_DATA_ZP
+                        }
+                    #endif
+                #else
+                    #ifdef SHOULD_USE_DATA_ZP
+                        if (input_on_padding)
+                            in[reg] = data_zp_val;
+                        else
+                    #endif
+                            in[reg] = *(__global PACKED_TYPE*)(conv_input + in_addr);
+                    in_addr += (INPUT0_SIZE_X + IWPAD) * 16;
+                 #endif
+            #else
+                #ifdef BLOCK_LOAD_INPUTS
+                    in[reg] = AS_PACKED_TYPE(intel_sub_group_block_read(&conv_input[in_addr]));
+                    #ifdef SHOULD_USE_DATA_ZP
+                        if (input_on_padding)
+                            in[reg] = data_zp_val;
+                    #endif
+                #else
+                    #ifdef SHOULD_USE_DATA_ZP
+                        if (input_on_padding)
+                            in[reg] = data_zp_val;
+                        else
+                    #endif
+                            in[reg] = AS_PACKED_TYPE(conv_input[in_addr]);// read SIMD_SIZE elements wide
+                #endif
+                in_addr += (INPUT0_SIZE_X + IWPAD);  // move to next row down
+            #endif
         }
-#endif
+
+        #ifdef BLOCK_LOAD_WEIGHTS
+            *((int8*)&w[0]) = as_int8(intel_sub_group_block_read8((const __global uint*) &weights[weight_addr]));
+            w[8]= as_int(intel_sub_group_block_read((const __global uint*) &weights[weight_addr + (SIMD_SIZE<<3)]));
+            weight_addr += SIMD_SIZE*NUM_FILTERS;
+        #else
+            for(int pf=0; pf < NUM_FILTERS; pf++) {
```

Comment:
In other cycles you use code style looks like "pf = 0" (with spaces before and after the symbol of equality).